pkg/security/secl/model/events.go

```diff
@@ -89,6 +89,17 @@ const (
 	UnshareMountNsEventType
 	// SyscallsEventType Syscalls event
 	SyscallsEventType
+	// AnomalyDetectionSyscallEventType Anomaly Detection Syscall event
+	AnomalyDetectionSyscallEventType
+	// AnomalyDetectionOpenEventType Anomaly Detection Open event
+	AnomalyDetectionOpenEventType
```

Comment:
I don't think we need a new event type for each anomaly detection: I believe the information that an event is an anomaly detection is already contained in the rule that the event matched (i.e. the anomaly detection custom rule). Keeping the event "as is" will also help simplify the code in `DispatchEvent`.
(I needed a different event type for syscalls because `SyscallsEvent` and `AnomalyDetectionSyscallEvent` are unmarshalled differently ... but I suspect that we'll actually be able to remove this if we move to the `Event.Flags` solution I mentioned earlier)